plutus-tx-plugin/plutus-tx-plugin.cabal

```diff
@@ -115,7 +115,7 @@ executable gen-plugin-opts-doc
 
   default-language: Haskell2010
 
-test-suite plutus-tx-tests
+test-suite plutus-tx-plugin-tests
```

Comment:
I changed this because `plutus-tx` contains `plutus-tx-test` and I was getting very confused.